api_module/src/Resources/data/install.sql

```diff
@@ -1 +1,6 @@
+-- Hamster (Doctrine entity)
 CREATE TABLE IF NOT EXISTS `PREFIX_hamster` (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(64) NOT NULL, description VARCHAR(64) NOT NULL, weight FLOAT NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB
+
+-- Horse (legacy ObjectModel)
+CREATE TABLE IF NOT EXISTS `PREFIX_horse` (id_horse INT AUTO_INCREMENT NOT NULL, name VARCHAR(64) NOT NULL, color VARCHAR(64) NOT NULL, weight FLOAT NOT NULL, PRIMARY KEY(id_horse)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB
+CREATE TABLE IF NOT EXISTS `PREFIX_horse_shop` ( `id_horse` int(10) unsigned NOT NULL, `id_shop` int(10) unsigned NOT NULL, PRIMARY KEY (`id_horse`, `id_shop`)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB
```

Comment:
You added a shop association table, but is it really needed? It doesn't seem your API can handle some specific multi-shop features, right? Even the ObjectModel itself doesn't seem to have multi shops fields defined